pwndbg/commands/context.py

```diff
@@ -278,21 +279,23 @@ def context_backtrace(frame_count=10, with_banner=True):
     return result
 
 
-def context_args():
+def context_args(with_banner=True):
```

Comment:
Is it ever called with `False`? Where?

Reply:
nope, but we use that for other things too and we can then call it if needed without having a banner inserted. I Think its not much harm to have a function option for that